### DISCUSSION TITLE
How can I see the connected clients to the server.

### DISCUSSION THREAD

**Discussion by Lichtmacher** [ltsp/ltsp#366] [Q&A]
I want to reboot the server and have to check if there are connected clients.
Just clients with loginscreens
or badlier clients with logged in users.

How can I check that. And maybe log out the users and shutdown the clients.

**Comment by alkisg** [comment]
Follow the LTSP installation instructions, which include the Epoptes client management tool, that can do all that and more.

https://ltsp.org/docs/installation/

**Comment by Lichtmacher** [comment]
Epoptes

I wanted to check it. But I didn't yet.

Thanks for all your work.
